lua/nvim-lsp-installer/platform.lua

```diff
@@ -25,6 +25,18 @@ M.is_unix = vim.fn.has "unix" == 1
 M.is_mac = vim.fn.has "mac" == 1
 M.is_linux = not M.is_mac and M.is_unix
 
+-- @return string @The libc found on the system, musl or glibc (glibc if ldd is not found)
+function M.get_libc()
```

Comment:
I wonder if this should be cached/memoized? There's a handy `memoize` function in `nvim-lsp-installer.data` for this!

Reply:
I bet it could. I was thinking of that, but didn't want to roll my own and make a mess in there, but since you provide a memoize function I can probably wrap it with it.